src/main/java/org/mockito/Mockito.java

```diff
@@ -95,15 +95,18 @@
  * <h3 id="0.1">0.1. <a class="meaningful_link" href="#mockito-android">Mockito Android support</a></h3>
  *
  * With Mockito version 2.6.1 we ship "native" Android support. To enable Android support, add the `mockito-android` library as dependency
- * to your project. This artifact is published to the same mockito organization:
+ * to your project. This artifact is published to the same Mockito organization and can be imported for Android as follows:
  *
  * <pre class="code"><code>
  * repositories { jcenter() }
- * dependencies { testCompile "org.mockito:mockito-android:2.+" }
+ * dependencies { androidTestCompile "org.mockito:mockito-android:2.+" }
```

Comment:
I think we should add the version 2.6 for android artefact, and add the mockit-core dependency otherwise, it is not imported by mockito-android.

```
androidTestCompile "org.mockito:mockito-core:2.6.+"
androidTestCompile "org.mockito:mockito-android:2.6.+"
```